packages/from-to-location-picker/src/index.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import LocationIcon from "@opentripplanner/location-icon";
+import { LocationPickerSpan, Button, FromToPickerSpan } from "./styled";
+
+const iconSize = "0.9em";
+
+const FromToLocationPicker = ({
+  fromText = "From here",
+  toText = "To here",
+  showIcons = true,
```

Comment:
These defaults should not be included here. The defaultProps are already sufficient.